tests/src/Core/Model/ProcessTest.php

```diff
@@ -16,13 +15,9 @@
  */
 class ProcessTest extends DatabaseTest
 {
-	/**
-	 * @var App
-	 */
-	private $app;
-
 	protected function setUp()
 	{
+		global $a;
```

Comment:
There's no more global `$a` variable and if there is, it's a bug, the app should now be retrieved using `BaseObject::getApp()`.

Reply:
I believe it's a forgotten fragment of the PR before the config-PR, I'll remove it :)

Reply:
The App holds a reference to the SmartyEngine indeed.